articles.tex

```diff
@@ -268,7 +268,7 @@
 An Advisor may resign by submitting in writing the reason for resignation to the Chairman.
 The resignation will be read at the following House Meeting and become effective at that time.
 \asubsection{Advisory Membership Term}
-Advisory Membership shall last until the member resigns.
+Advisory Membership shall last until the Advisor no longer meets the qualifications as defined in \ref{Advisory Membership Qualifications} or until the Advisor resigns as defined in \ref{Advisory Membership Resignations}, at which time their membership is revoked.
```

Comment:
I think you can skip the `as defined in .... revoked.` 
The current wording could be construed to mean that when and advisor resigns or leaves RIT they also lose their existing "normal" membership.